react-native-libraries.json

```
@@ -12060,5 +12060,26 @@
   ],
   "ios": true,
   "android": true
+},
+  {
+  "githubUrl": "https://github.com/kimolalekan/react-native-custom-select",
+  "npmPkg": "react-native-custom-select",
+  "examples": [
+    "https://github.com/kimolalekan/react-native-custom-select/blob/main/examples/index.js",
+    "https://snack.expo.dev/@geek_lekan/react-native-custom-select-example"
+  ],
+  "images": ["https://raw.githubusercontent.com/kimolalekan/react-native-custom-select/main/screenshot-image.png"],
+  "ios": true,
+  "android": true,
+  "web": true,
+  "expoGo": true,
+  "windows": false,
+  "macos": false,
+  "tvos": false,
+  "visionos": false,
+  "unmaintained": false,
+  "dev": false,
+  "template": false,
+  "newArchitecture": true
```

Hello @kimolalekan, thanks for adding new entry to the directory! 👍 

```suggestion
  "githubUrl": "https://github.com/kimolalekan/react-native-custom-select",
  "examples": [
    "https://github.com/kimolalekan/react-native-custom-select/blob/main/examples/index.js",
    "https://snack.expo.dev/@geek_lekan/react-native-custom-select-example"
  ],
  "images": ["https://raw.githubusercontent.com/kimolalekan/react-native-custom-select/main/screenshot-image.png"],
  "ios": true,
  "android": true,
  "web": true,
  "expoGo": true,
  "newArchitecture": true
```